scripts/handle-user-data

```diff
@@ -35,11 +35,12 @@ mount_data_partition() {
     mkdir -p /mnt/$PARTNAME/var/lib/boot2docker/etc/docker
     ln -s /mnt/$PARTNAME/var/lib/boot2docker/etc/docker /etc/docker
 
-    # Here we used bind mount because currently openshift doesn't work
```

Comment:
this comment is now obsolete? So we have a single partition now? No need for mount bind? What's about the Docker config and images?